ant-networking/src/lib.rs

```diff
@@ -728,6 +730,28 @@ impl Network {
         Ok(None)
     }
 
+    /// Request the node version of a peer on the network.
+    pub async fn get_node_version(&self, peer_id: PeerId) -> Result<Version, String> {
+        let request = Request::Query(Query::GetVersion(NetworkAddress::from_peer(peer_id)));
+        match self
+            .send_request(request, peer_id, Default::default())
```

Comment:
node only try to fetch version from near range peers, which shall sit in the RT and swarm driver can figure out peer's multi_addr internally.

However, for client, in most case, the targeted node is not in the RT, hence the multi_addr is not available.
Sending a `default` here will result in the request failed to be sent out in most cases.
which result in a wrong interception of nodes' version.